### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by michaeltoohig**

### CODE REVIEW

**Review by kautea — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-!DOCTYPE html>
+<!DOCTYPE html>
```
Comment (michaeltoohig) [inline index.html:1]: I added < here because it seems there was a typo.

File: index.html [hunk 2]
```diff
@@ -37,4 +37,4 @@ <h1>Chapitre 1 : Les éléments qui composent un ordinateur</h1>
   <script src="js/main.js"></script>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (michaeltoohig) [inline index.html:40]: I added a newline to the end of the file. Well my editor did it automatically actually.

File: index.html (hunk 1 above)
Comment (kautea) [reply index.html:1]: okey nice wan

File: index.html (hunk 2 above)
Comment (kautea) [reply index.html:40]: okey thanks

**Pull request merged**